rtv/main.py

```diff
@@ -94,6 +95,9 @@ def main():
                 page = SubredditPage(stdscr, reddit, args.subreddit)
                 page.loop()
 
+    except InvalidUserPass:
+        print('Invalid password for username: ' + args.username)
```

Comment:
Sorry to be nitpicky, but please change the formatting style to match the rest of the errors.

See https://docs.python.org/2/library/string.html#format-examples for examples of new-style string formatting.